lib/util/distance.js

```diff
@@ -1,7 +1,7 @@
-export function distanceStringImperial (meters) {
+export function distanceStringImperial (meters, abbreviate) {
```

Comment:
Consider replacing this file with [humanize-distance](https://www.npmjs.com/package/humanize-distance).